NugetLightRequest.cs

```diff
@@ -85,6 +85,7 @@ public override SecureString CredentialPassword
         internal const string DefaultConfig = @"<?xml version=""1.0""?>
 <configuration>
   <packageSources>
+    <add key=""nuget.org"" value=""https://api.nuget.org/v3/index.json"" protocolVersion=""3"" />
```

Comment:
Has this been verified to not have any impact on PowerShellGet v2 which relies on this provider?

Reply:
I was looking for some tests to validate the scenarios.  What's the easiest way to validate? There is varying docs on what is now included in PS6+ versus < 5 etc. and I'm not sure what the best way to look at the matrix is here (e.g., PS7.2 includes the provider)

Reply:
Honestly, OneGet isn't being maintained and effectively deprecated.  So this might be more effort that it's worth.  Maybe be better to have this issue opened as a doc bug for PackageManagement.

Reply:
Not knowing what the component structure is or where "OneGet" gets exposed to customers, I can't comment accurately on this, but it does feel like the default experience is broken for `Install-Package` and the like in some circumstances.  Unfortunately the resulting error messages wouldn't lead to any logical PackageManagement documentation point either.  This transient aspect results in breaking OTHER scenarios as well, namely `dotnet build` and other CLI-based operations for `dotnet`

Reply:
@alerickson We should not be using `powershell-rel` in public.  Can we discuss why this is needed and come up with an alternative?

Reply:
FWIW, this appears to be affecting a bunch of Azure Pipelines and GitHub Actions customers: https://github.com/actions/virtual-environments/issues/5768